managed/CounterStrikeSharp.API/Core/Model/CUtlSymbolLarge.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Reflection.Metadata;
```

Comment:
not to be picky, just trying to keep things clean (we have messed up already haha)
is this referenced?